rule/expr.go

```diff
@@ -237,6 +237,46 @@ func (n *exprIn) Eval(params Params) (*Value, error) {
 	return BoolValue(false), nil
 }
 
+type exprGt struct {
```

Comment:
I strongly suggest we use `GT` rather than `Gt` to remove the need to rename when when moving to 0.0.7.

Reply:
Done https://github.com/heetch/regula/pull/103/commits/35f736165459179b6bce32af3867a38ec6d8ec07